app-text/xapers/xapers-0.8.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6})
+
+inherit distutils-r1
+
+DESCRIPTION="Xapers is a personal document indexing system."
```

Comment:
no package name in the `DESCRIPTION` and no dot too

Reply:
Fixed